packages/extension/src/config.ts

```diff
@@ -1634,6 +1634,7 @@ export const EmbedChainInfos: ChainInfo[] = [
     restConfig: EVMOS_REST_CONFIG,
     chainId: "evmos_9001-1",
     chainName: "Evmos (Beta)",
+    coinType: 60,
```

Comment:
The coinType field in `ChainInfo` type is a temporary field that has been added to support ENS. There’s a lot of room for improvement, thus having high possibility if being deprecated as well. Thereby I would recommend using bip44 field instead of using this field